cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/includes/bottom.jsp

```diff
@@ -23,9 +23,9 @@
     <div class="row" style="text-align: center;">
         <hr>
         <spring:eval var="osfInstitutionLoginUrl" expression="@casProperties.getProperty('osf.institutionLogin.url')" />
-        <a id="institution-login" href="${osfInstitutionLoginUrl}${not empty param.service ? '&redirect_url=' : ''}${fn:escapeXml(param.service)}">Login Through Your Institution</a>&nbsp;&nbsp;
+        <a id="institution-login" href="${osfInstitutionLoginUrl}${not empty param.service ? '&redirect_url=' : ''}${fn:escapeXml(param.service)}">Login&nbsp;Through&nbsp;Your&nbsp;Institution</a>&nbsp;&nbsp;
```

Comment:
this is fine but if `&nbsp;` ever gets out of hand you can also use the css style.

http://www.w3schools.com/cssref/pr_text_white-space.asp